### DISCUSSION TITLE
Is there a way to trigger new plots based on thread count ?

### DISCUSSION THREAD

**Discussion by gdias1992** [swar/Swar-Chia-Plot-Manager#594] [Q&A]
I would like to control when new Plots are triggered based on thread count.

```
IF(
        (FREE_THREAD_COUNT > X) && 
        (CURRENT_IN_PROGRESS_PLOTS < MAX_CONCURRENT) &&
        (CURRENT_IN_PROGRESS_PLOTS_PHASE_1 < MAX_FOR_PHASE_1)
){
          START NEW PLOT
}
```

To get the thread count you have to consider how many Plots are in progress and it's phase.

Is it possible ?

**Comment by NickHuang0902** [comment]
I also want the function. Because phase_1 used more thread and phase_2~5 only one thread.
global:
    max_concurrent
    max_thread

**Comment by NickHuang0902** [comment]
Or add global parameter max_AllPase1.
global:
max_concurrent
max_AllPase1
